sphinxcontrib/inmanta/pygments_lexer.py

```diff
@@ -148,21 +105,21 @@ def process_id(self, match):
                 ),
             ),
             (in_class_name, Name.Class),
-            ('["]{3}([\\n]|.)*?["]{3}', Comment.Multiline),  # t_begin_mls
+            (r'["]{3}([\\n]|.)*?["]{3}', Comment.Multiline),  # t_begin_mls
```

Comment:
I believe this one is not the same as it was before.
```suggestion
            (r'["]{3}([\n]|.)*?["]{3}', Comment.Multiline),  # t_begin_mls
```

Reply:
thx!